test/inspector/inspector-helper.js

```diff
@@ -169,13 +169,13 @@ TestSession.prototype.processMessage_ = function(message) {
     assert.strictEqual(id, this.expectedId_);
     this.expectedId_++;
     if (this.responseCheckers_[id]) {
-      assert(message['result'], JSON.stringify(message) + ' (response to ' +
-             JSON.stringify(this.messages_[id]) + ')');
+      assert(message['result'], `${JSON.stringify(message)} (response to ${
+                                   JSON.stringify(this.messages_[id])})`);
```

Comment:
I've created some new variables if this improved many places per file. But I also tried not to be too creative to not overload the diff.